tests/AgDatabaseMove.Integration/TestAgDatabase.cs

```diff
@@ -0,0 +1,75 @@
+﻿using AgDatabaseMove.Integration.Fixtures;
+using AgDatabaseMove.SmoFacade;
+using System.Collections.Generic;
+using Xunit;
+using Smo = Microsoft.SqlServer.Management.Smo;
+
+namespace AgDatabaseMove.Integration
+{
+  public class TestAgDatabase : IClassFixture<TestAgDatabaseFixture>
+  {
+    private readonly TestAgDatabaseFixture _testAgDatabaseFixture;
+
+    public TestAgDatabase(TestAgDatabaseFixture testAgDatabaseFixture)
+    {
+      _testAgDatabaseFixture = testAgDatabaseFixture;
+    }
+
+    private AgDatabase AgDatabase => _testAgDatabaseFixture._agDatabase;
+
+    private string LoginName => _testAgDatabaseFixture._loginName;
+
+    private string LoginPassword => _testAgDatabaseFixture._loginPassword;
+
+    private string DefaultDatabase => _testAgDatabaseFixture._loginDefaultDatabase;

```

Comment:
What benefit does this provide?  All of these values can be accessed from within the class already